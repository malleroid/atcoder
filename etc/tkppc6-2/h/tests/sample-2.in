4
17 59
59 91
78 71
24 19
3
1 3
2 4
2 3
