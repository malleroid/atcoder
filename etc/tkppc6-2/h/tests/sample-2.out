186
188
169
